sample-code/manifest.yaml

```diff
@@ -0,0 +1,10 @@
+applications:
```

Comment:
[q] Is this still needed?

Reply:
It's not, but this would allow us to more easily deploy it to CF, if we want to. With the downside of exposing a route and service name that are meaningless outside of our testing setup.

But because the file contains nothing specific about the AI SDK I guess it would be better to remove it.

Reply:
removed it for now, we can always add it back in if we like